all-actors/src/main/java/org/sunbird/cert/actor/CertificateGeneratorActor.java

```diff
@@ -96,40 +97,46 @@ private void generateCertificate(Request request) throws BaseException {
         }
         String orgId = (String) ((Map) request.get(JsonKey.CERTIFICATE)).get(JsonKey.ORG_ID);
         String tag = (String) ((Map) request.get(JsonKey.CERTIFICATE)).get(JsonKey.TAG);
+        directory = "conf/" + orgId.concat("_") + tag.concat("_") + htmlTempalteZip.getZipFileName().concat("/");
         List<Map<String, String>> certUrlList = new ArrayList<>();
         for (CertModel certModel : certModelList) {
             String certUUID = "";
             try {
                 certUUID = certificateGenerator.createCertificate(certModel, htmlTempalteZip, (String) ((Map<String, Object>) ((Map<String, Object>) request.getRequest().get(JsonKey.CERTIFICATE)).get(JsonKey.KEYS)).get(JsonKey.ID));
             } catch (Exception ex) {
-                cleanup();
-                logger.error("CertificateGeneratorActor:generateCertificate:Exception Occurred while generating certificate.", ex);
+                cleanup(directory, certUUID);
+                logger.error("CertificateGeneratorActor:generateCertificate:Exception Occurred while generating certificate. : " + ex.getMessage());
                 throw new BaseException("INTERNAL_SERVER_ERROR", IResponseMessage.INTERNAL_ERROR, ResponseCode.SERVER_ERROR.getCode());
             }
-            certUrlList.add(uploadCertificate(certUUID, certModel.getIdentifier(), orgId, tag));
+            certUrlList.add(uploadCertificate(certUUID, certModel.getIdentifier(), orgId, tag, directory));
+            cleanup(directory, certUUID);
         }
         Response response = new Response();
         response.getResult().put("response", certUrlList);
         sender().tell(response, getSelf());
-        cleanup();
         logger.info("onReceive method call End");
     }
 
-    private void cleanup() {
+    private void cleanup(String path, String fileName) {
         try {
-            File file = new File("conf/certificate");
-            FileUtils.deleteDirectory(file);
+            File directory = new File(path);
+            File[] files = directory.listFiles();
+            for (File file : files) {
+                if (file.getName().startsWith(fileName)) file.delete();
+            }
+            File file = new File(path);
+            file.delete();
```

Comment:
is the directory getting deleted? I thought we didn't want to do this. Pls remove 127 and 128.